core/src/raw/http_util/header.rs

```diff
@@ -199,6 +200,21 @@ pub fn parse_content_disposition(headers: &HeaderMap) -> Result<Option<&str>> {
     }
 }
 
+/// Parse header value to string according to name.
+pub fn parse_header_to_str<K: AsHeaderName>(headers: &HeaderMap, name: K) -> Result<Option<&str>> {
```

Comment:
Would you like to refactor APIs like `parse_content_disposition` at the same time? We can do this in next PR.